src/app/services/sms/sms.service.ts

```diff
@@ -50,6 +51,9 @@ export const twilioClientCache = new NodeCache({
   deleteOnExpire: true,
   stdTTL: 10,
 })
+const ddClient = statsdClient.childClient({
+  prefix: 'vendor.twilio.',
```

Comment:
nit: should we extract this? wary of magic strings. Also suggest renaming to `DD_CLIENT` since it's a top-level constant

Reply:
Right, let me do that. I'll extract a twilioDDClient somewhere that include the fixed prefix so we can avoit the repetition.

I hesitated to capitalize the name but I didn't because I typically only capitalize primitive constants, not objects or instances. And I think we do the same in other parts of the code? Like `logger` or  `secretsManager` are not capitalized either?

Reply:
https://github.com/opengovsg/FormSG/pull/3824/commits/22335bccba9c58f8b91685a45bf12a7b62f0c28b